packages/storybook/package.json

```diff
@@ -32,6 +32,7 @@
     "@nrwl/devkit": "*",
     "@nrwl/linter": "*",
     "@nrwl/workspace": "*",
+    "@storybook/core": "^6.0.21",
```

Comment:
@juristr where is this supposed to come from? I don't think directly depending on v6 makes sense for v5 projects right? Is this assumed to be part of like `@storybook/angular` or something?

Reply:
No, we cannot depend on Storybook directly. Let me check this tomorrow

Reply:
obviously `@nrwl/storybook` depends on the core package (e.g. [here](https://github.com/nrwl/nx/blob/master/packages/storybook/src/executors/storybook/storybook.impl.ts#L5), but due to the current situation we cannot really reference it in here, since we support/generate both, v5 storybook version as well as v6.

@bekos if you don't add this here, do e2e tests fail?

Reply:
Although I am sure that I added it because something was failing, after rebasing and running tests without it, seems that now it doesn't 😕 Problem solved 😅